### ISSUE TITLE
Start OsmAnd in Following mode

### ISSUE THREAD

**Issue by alyonapinchuk** [osmandapp/Osmand#6151]
Please add a setting to start OsmAnd in the following mode for different app profiles. It is not handy for motorbike users to tap the location button during the ride in the map view mode.

**Comment by Bushmills** [comment]
Definitely! This has been (no, it still is...) one of the gripes I have with OsmAnd. It also doesn't make much sense, to provide an option ("follow") which, when enabled, requires additional interaction to become effective. There's no need for a new option - the existing option will do nicely, if done properly.

**Comment by vshcherb** [comment]
It is planned

**Comment by Bushmills** [comment]
Good to hear. When looking at its implementation, please put also attention to that it's currently possible to lose following, when GPS reception drops out temporarily, and actual position differs from last known or estimated position far enough to highlight the cross hair icon. I might call this a bug,and is presumably addressed by the same change,

**Comment by scaidermern** [comment]
Would be really nice to have this implemented in an upcoming release.
